packages/terra-dialog-modal/src/terra-dev-site/doc/example/DialogModalWithFocus.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import Button from 'terra-button';
+import ActionHeader from 'terra-action-header';
+import ActionFooter from 'terra-action-footer';
+import AppDelegate from 'terra-app-delegate';
+import DialogModal from '../../../DialogModal';
+
+const propTypes = {
+  app: AppDelegate.propType,
+};
+
+class DialogModalWithFocus extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isOpen: false,
+    };
+
+    this.handleOpenModal = this.handleOpenModal.bind(this);
+    this.handleCloseModal = this.handleCloseModal.bind(this);
+  }
+
+  handleOpenModal() {
+    this.setState({ isOpen: true });
+  }
+
+  handleCloseModal() {
+    this.setState({ isOpen: false });
+  }
+
+  render() {
+    const paraOne = ['Lorem ipsum dolor sit amet, consectetur adipiscing elit. ',
+      'Maecenas molestie in lorem vel facilisis. Quisque ac enim nec lectus malesuada faucibus.',
+      'Integer congue feugiat ultricies.',
+      ' Nunc non mauris sed tellus cursus vestibulum nec quis ipsum.',
+      'Vivamus ornare magna justo, et volutpat tortor congue ut. Nulla facilisi.',
+      ' Cras in venenatis turpis. Nullam id odio justo. Etiam vehicula lectus vel purus consectetur cursus id sit amet diam.',
+      'Donec facilisis dui non orci hendrerit pharetra. Suspendisse blandit dictum turpis, in consectetur ipsum hendrerit eget.',
+      'Nam vehicula, arcu vitae egestas porttitor,',
+      'turpis nisi pulvinar neque, ut lacinia urna purus sit amet elit.'];
+    return (
+      <div>
+        <DialogModal
+          ariaLabel="Default Dialog Modal"
+          isOpen={this.state.isOpen}
+          onRequestClose={this.handleCloseModal}
+          header={<ActionHeader title="Action Header used here" onClose={this.handleCloseModal} />}
+          footer={<ActionFooter start="Footer Goes here" />}
```

Comment:
(2 of 3) Let's update the footer example text to say "Action Footer used here" to match what is in header and it is clear to use the Action Footer.